package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cloudproof_js",
-  "version": "6.0.0",
+  "version": "7.0.0",
```

Comment:
Why a breaking change?

Reply:
There is no backward compatibility for keys.